integrations/symfony-bundle.rst

```diff
@@ -91,7 +91,8 @@ You can configure your clients with default options. These default values will b
             acme:
                 factory: 'httplug.factory.curl'
                 config:
-                    78: 4 #CURLOPT_CONNECTTIMEOUT
+                    CURLOPT_CONNECTTIMEOUT: 4
+                    CURLOPT_SSL_VERIFYHOST: 0
```

Comment:
could we use true or false for verifyhost? feels very odd to use 0 and 1. even if the underlying C library probably does that.